pages/governance.md

```diff
@@ -0,0 +1,110 @@
+
+JuMP is a [fiscally sponsored project of NumFOCUS](https://numfocus.org/project/jump),
+a 501(c)(3) public charity in the United States. NumFOCUS help the JuMP project
+by providing legal and financial support.
+
+JuMP has a Steering Council that acts as the interface between the community and
+NumFOCUS. The main role of the Steering Council is to:
+
+* Approve expenditures related to JuMP, e.g., the renewal of our [jump.dev](https://jump.dev)
+  domain name.
+* Negotiate and approve contracts between NumFOCUS and external contractors who
+  provide paid work to the JuMP project.
+* Represent JuMP in interactions with NumFOCUS, e.g., by attending NumFOCUS's
+  annual summit.
+
+The current members of the steering committee are:
+
+* [Miles Lubin (Google)](https://github.com/mlubin)
+* [Juan Pablo Vielma (Google)](https://github.com/juan-pablo-vielma)
+* [Joey Huchette (Rice University)](https://github.com/joehuchette)
+* [Oscar Dowson](https://github.com/odow)
+* [Changhyun Kwon (U. South Florida)](https://github.com/chkwon)
+
+If a member of the steering committee leaves the committee, the remaining
+members will invite a member of the community to join in order to maintain a
+quorum of five members.
+
+### Conflict of interest
+
+It is expected that Steering Council members will be employed at a wide range of
+companies, universities and non-profit organizations. Because of this, it is
+possible that Members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of the Project that may influence their work on the Project.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+
+All members of the Council shall disclose any conflict of interest they may
+have. Members with a conflict of interest in a particular issue may participate
+in Council discussions on that issue, but must recuse themselves from voting on
+the issue.
+
+## Channels of community involvement
+
+There are four main channels that the JuMP project uses to engage with the
+community.
+
+### Discourse forum
+
+The [Discouse forum](https://discourse.julialang.org/c/domain/opt/13) is the
+best place to go for help if you are a user of JuMP.
+
+Before posting a question, we ask that you:
+
+* Search the forum for similar questions that may have been already answered.
+* Read the first post of [PSA: make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757/81).
+
+Remember to post a simplified version of your problem that demonstrates the
+error you are experiencing or the code you currently have, to format your code
+properly, and to provide as much detail as possible.
+
+### Gitter chatroom
+
+The [Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev) is for
+developer-focused chat about JuMP. If you are hacking on JuMP and have a
```

Comment:
Also doesn't fit in with the governance structure, unless it's a statement like "the council will maintain a developer chat room ...".